site/src/constants.ts

```diff
@@ -22,13 +22,13 @@ export const healthAttributeMap = [
   },
   {
     id: "latestDesign",
-    positive: "Matches latest design",
-    negative: "Doesn’t match latest design",
+    positive: "UI Kit parity",
+    negative: "No UI Kit parity",
```

Comment:
nit/ We use the same language on the health page and for our OKRs so they match. If this is being updated, the OKR metric should match.

Reply:
Love me a good nit pickin' ;) That's a good point, fyi I am trying to reword/optimize our OKRs right now and will make sure it says the same thing. Cheers.

Reply:
If you want another nit: to me, "UI Kit parity" would mean latest design _and_ has all variants.